### ISSUE TITLE
Methods returning task are incorrectly flagged as non-async

### ISSUE THREAD

**Issue by NeilMacMullen** [priyanshu92/AsyncMethodNameFixer#5]
### Installed product versions
- Visual Studio: 15.7.4 
- This extension: 1.0

In interface code it is not possible to use the 'async' keyword because there is no method body.  So interface signatures have to look like....

`Task<ImmutableArray<T>> SimpleQueryAsync<T>(`

The extension falsely flags these signatures as "having Async at the end" and offers to remove the Async.  

A related issue is that in code such as  
```
async Task<int> InnerAsync() {await Task.Delay(1);}
Task<int> WrapperAsync() {return InnerAsync();}

```
the WrapperAsync method will be incorrectly flagged.  

Obviously the suppression pragmas can be used to avoid the warning but it would be nice if these weren't needed in these cases.

**Comment by NeilMacMullen** [comment]
Just realised this is a duplicate of https://github.com/priyanshu92/AsyncMethodNameFixer/issues/2

**Comment by priyanshu92** [comment]
Fixed in #6
